modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/metal/js/components/RuleEditor/RuleEditor.es.js

```diff
@@ -277,6 +295,7 @@ class RuleEditor extends Component {
 		strings: {
 			value: {
```

Comment:
Maybe we can declare this structure as we are declaring the other props structures. Something like: 
```
strings: Config.shapeOf({
 value: config.object()
}).val({
     value: {
         ...
     }
})
```